generators/server/templates/_pom.xml

```diff
@@ -941,9 +963,6 @@
                     Enable the line below to have remote debugging of your application on port 5005
                     <jvmArguments>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
                     -->
-                    <arguments>
-                        <argument>--spring.profiles.active=dev</argument>
```

Comment:
I'm not sure what you mean... None of the static assets (which are served from different location depending on profile) contain any placeholders themselves, if that's what you were getting at?